src/schema.js

```diff
@@ -4,53 +4,85 @@ const DEFAULT_LABEL = "…"
 
 const blocks = {
 	api: {
-		parents: ['server'],
+		parents: ['environment', 'server'],
```

Comment:
Unfortunately, this renders the completion almost useless. With `environment` as every blocks/attributes parent, the completion always suggests *all* blocks/attributes as possible children for `environment`:

![broken_completion](https://user-images.githubusercontent.com/1311513/175886491-22c0acf7-280e-4281-912e-64841436832d.gif)
ironment`.